src/modules/Gamma/info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Gamma",
+    "description": "Apply gamma correction on the image",
+    "inputs": {
+        "adjustment": {
+            "type": "float",
+            "desc": "gamma correction for the new image",
```

Comment:
Just to be extra clear let's also mention this is inverse of the actual gamma.

Reply:
I will change the module name to Gamma Correction and also add a link to Wikipedia's page on Gamma to the description. Thanks!

I didn't quite get 'inverse of the actual gamma'. I think it is gamma, how is this inverse of the actual gamma?

Reply:
Oh Actually the gamma correction is the inverse of the actual gamma factor, so just to be extra clear I was thinking maybe we should mention that this is the gamma correction, inverse of the actual gamma factor. Thanks!

Reply:
Oh okay, I will do that!